test/0.8.25/vaults/dashboard/dashboard.test.ts

```diff
@@ -54,10 +56,11 @@ describe("Dashboard", () => {
     weth = await ethers.deployContract("WETH9__MockForVault");
     wsteth = await ethers.deployContract("WstETH__HarnessForVault", [steth]);
     hub = await ethers.deployContract("VaultHub__MockForDashboard", [steth]);
+    lidoLocator = await ethers.deployContract("LidoLocator__HarnessForDashboard", [steth, wsteth]);
```

Comment:
```suggestion
    lidoLocator = await deployLidoLocator({ lido: steth, wstETH: wsteth });
```

See the comment above for the harness locator contract removal

Reply:
fixed